src/offerings.ts

```diff
@@ -109,8 +112,38 @@ export interface PurchasesStoreProduct {
      * Note: Not available for Amazon.
      */
     readonly subscriptionPeriod: string | null;
+    /**
+     * Default subscription option for a product. Google Play only.
+     */
+    readonly defaultOption: SubscriptionOption | null;
```

Comment:
Should we update the docs of the other fields here indicating that they will contain information of the defaultOption in Google's subscriptions? For example `price` or `currencyCode`

Reply:
Ohh, good call! 👍